### PR TITLE
PPTP-1187 - View Group Members (Org List) page

### PR DESCRIPTION
**Pull request by SteveSugden**
#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [ ] User Acceptance Tests (UAT) were run locally and have passed.
 - [x] No Accessibility errors/warnings reported by Wave

image

The alert above is because both remove links go to the same "null" location at present

### CODE REVIEW

File: test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationListControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+      "user is not authorised" in {
+        unAuthorizedUser()
+        val result = controller.displayPage()(getRequest())
+
+        intercept[RuntimeException](status(result))
+      }
+    }
+
+    "redirects to registration page" when {
```
Comment (RyanKellyHMRC) [inline test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationListControllerSpec.scala:112]: task list?

**Review by RyanKellyHMRC — APPROVED** [review]
Some potential naming change suggestions. Not a show stopper.

File: app/assets/stylesheets/_registration.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.hmrc-add-to-a-list--short .hmrc-add-to-a-list__identifier {
+  width: 52%;
```
[the remark this answers is not in the document]
Comment (SteveSugden) [reply app/assets/stylesheets/_registration.scss:2]: To make it look like the prototype.  Without it the "name" column wrapped (i think from memory).  I'll take another look.  I don't like making css changes really.

File: app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationListController.scala [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+import uk.gov.hmrc.plasticpackagingtax.registration.forms.group.AddOrganisation
+import uk.gov.hmrc.plasticpackagingtax.registration.forms.group.AddOrganisation.form
+import uk.gov.hmrc.plasticpackagingtax.registration.models.registration.Cacheable
+import uk.gov.hmrc.plasticpackagingtax.registration.models.request.JourneyAction
+import uk.gov.hmrc.plasticpackagingtax.registration.views.html.group.organisation_list
+import uk.gov.hmrc.plasticpackagingtax.registration.{controllers => pptControllers}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+
+@Singleton
+class OrganisationListController @Inject() (
```
[the remark this answers is not in the document]
Comment (SteveSugden) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationListController.scala:38]: "Group" is already implied in the package name.  Names are hard!

File: app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationListController.scala [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+  def displayPage(): Action[AnyContent] =
+    (authenticate andThen journeyAction) { implicit request =>
+      request.registration.groupDetail.fold(
+        Redirect(pptControllers.routes.RegistrationController.displayPage())
+      )(
+        detail =>
+          if (detail.members.isEmpty)
+            // Must add at least one group member to use this page
+            Redirect(pptControllers.group.routes.OrganisationDetailsTypeController.displayPage())
```
[the remark this answers is not in the document]
Comment (SteveSugden) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationListController.scala:54]: This is redirecting to Kiran's "add first group member" page

File: test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationListControllerSpec.scala (hunk 1 above)
Comment (SteveSugden) [reply test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationListControllerSpec.scala:112]: I'll re-organise these tests into "display" and "submit" tests.  There should be one display direct when the GroupDetails aren't present and a different one when the GroupMembers list is empty.

File: app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationListController.scala (hunk 3 above)
[the remark this answers is not in the document]
Comment (RyanKellyHMRC) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationListController.scala:38]: We did talk about and agree that the "group" context should be sufficient. If only I had a memory!

File: app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationListController.scala (hunk 4 above)
[the remark this answers is not in the document]
Comment (RyanKellyHMRC) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationListController.scala:54]: Missed the group context, again!

**Pull request merged**